README.md

```diff
@@ -1,9 +1,66 @@
 Openmeeg website
 ================
 
-The source code of the [openmeeg website](<http://openmeeg.github.io/>)
+This repository contains the source code of the [openmeeg website](<http://openmeeg.github.io/>)
+which contains all the documentation to get started in using
+[openmeeg](<https://github.com/openmeeg/openmeeg>).
 
-requirements:
+The documentation is written in [reStructuredText](http://docutils.sourceforge.net/rst.html)
+and build using [**Sphinx**](http://www.sphinx-doc.org/) to render as a website.
+
+### Requirerments 
+
+In order to build our documentation, the following packages need to be available:
 - sphinx
-- texlive-anyfontsize
 - pdflatex
+- texlive-anyfontsize
+- make
+
+Refer to **Sphinx** doc in [how to install it](http://www.sphinx-doc.org/en/stable/install.html),
+ but in short you can install it from [**pip**](https://pypi.python.org/pypi/pip) 
+(but you can also get from most package managers or conda).
+
+```sh
+$ pip install sphinx
+```
+
+Use your favorite package manager to install the **latex**. For Debian based
```

Comment:
say that latex is optional as by default you do : make html
and it does not need latex.